src/helpers/authentication.js

```diff
@@ -85,30 +85,37 @@ module.exports = (application) => {
   };
 
   const refreshToken = (user) => {
-    if (user === 'default') return;
     if (isTest) return;
 
     clearTimeout(timeouts[user]);
 
     const token = tokens[user];
     const createdAt = toMilliseconds(token.createdAt);
     const expiresIn = toMilliseconds(tokens[user].expiresIn);
-    const expiresAt = createdAt + expiresIn;
+    const expiresAt = createdAt + expiresIn - 600_000; // refresh 10 minutes before expected expiry
```

Comment:
Why move it up here? Seems like it was more accurate in terms of variable naming before.

Reply:
I would like to print `Token will be refreshed at ${new Date(expiresAt)}.` to the console.. if we don't move `- 600_000` up here, we would need to do it twice when printing, and a 2nd time when calculating `setTimeout` delay..